### ISSUE TITLE
`count_with_total()` fails when column to be totalled is numeric

### ISSUE THREAD

**Issue by ccsarapas** [ccsarapas/lighthouse#10]
``` r
library(lighthouse)

mtcars %>% 
  count_with_total(am, vs)
#> Error in `dplyr::bind_rows()`:
#> ! Can't combine `..1$am` <double> and `..2$am` <character>.
```

<sup>Created on 2024-04-12 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

**Comment by ccsarapas** [comment]
Or any type other than character or factor.

**Comment by ccsarapas** [comment]
Closed by #13.
